### PR TITLE
NO-ISSUE: Fix standalone job in the release workflow

### PR DESCRIPTION
**Pull request by caponetto**
Fixes https://github.com/kiegroup/kie-tools/actions/runs/6485901985/job/17615030875

Needs cherry-pick to `0.32.0-prerelease` branch

### CODE REVIEW

File: .github/workflows/release_build.yml [hunk 1]
```diff
@@ -908,16 +908,16 @@ jobs:
         run: |
           echo "Copy standalone resources"
           rm -rf $STANDALONE_EDITORS_DIR
-          mkdir $STANDALONE_EDITORS_DIR
+          mkdir $STANDALONE_EDITORS_DIR $STANDALONE_EDITORS_DIR/bpmn $STANDALONE_EDITORS_DIR/dmn $STANDALONE_EDITORS_DIR/swf
```
Comment (yesamer) [inline .github/workflows/release_build.yml:911]: @caponetto Perfect, thank you 👍 
I forgot to create those directories :/
Apologies for that.

File: .github/workflows/release_build.yml [hunk 2]
```diff
@@ -908,16 +908,16 @@ jobs:
...
           cp ${{ github.workspace }}/kie-tools/packages/serverless-workflow-standalone-editor/dist/swf/index.js $STANDALONE_EDITORS_DIR/swf/
           rm -f ./standalone/bpmn/index.js
           rm -f ./standalone/dmn/index.js
           rm -f ./standalone/swf/index.js
-          ln -s ../${{ inputs.tag }}/bpmn/index.js ./standalone/bpmn/index.js
-          ln -s ../${{ inputs.tag }}/dmn/index.js ./standalone/dmn/index.js
-          ln -s ../${{ inputs.tag }}/swf/index.js ./standalone/swf/index.js
+          ln -s $STANDALONE_EDITORS_DIR/bpmn/index.js ./standalone/bpmn/index.js
+          ln -s $STANDALONE_EDITORS_DIR/dmn/index.js ./standalone/dmn/index.js
+          ln -s $STANDALONE_EDITORS_DIR/swf/index.js ./standalone/swf/index.js
```
Comment (yesamer) [inline .github/workflows/release_build.yml]: @caponetto Why is this change necessary? That generates a broken symlink in my local.

**Review by yesamer — COMMENTED** [review]
Thank you @caponetto for catching that. Just a question.

File: .github/workflows/release_build.yml (hunk 2 above)
Comment (caponetto) [reply .github/workflows/release_build.yml]: Hey @yesamer,
Considering that the working directory is `${{ github.workspace }}/kogito-online`, 
`../${{ inputs.tag }}/bpmn/index.js` goes back one dir and try to access `${{ inputs.tag }}/bpmn/index.js`.
It didn't make much sense here. Am I missing something?

Comment (yesamer) [reply .github/workflows/release_build.yml]: Why it doesn't make sense? That is the expected behaviour.
Eg. `standalone/dmn/index.js` should point to the latest version of the dmn js, which is in `standalone/0.31.0/dmn/index.js`.

Screenshot 2023-10-12 at 12 40 48

In addition, that is the current behaviour we have in place --> https://github.com/kiegroup/kogito-online/blob/gh-pages/standalone/dmn/index.js

Comment (caponetto) [reply .github/workflows/release_build.yml]: Yep, that's exactly the reason I changed it to `$STANDALONE_EDITORS_DIR/{bpmn|dmn|swf}/index.js`.
It points to `${{ github.workspace }}/kogito-online/standalone/${{ inputs.tag }}/{bpmn|dmn|swf}/index.js`.

Comment (yesamer) [reply .github/workflows/release_build.yml]: I applied your changes in my local, can you please confirm that is the behaviour you want to introduce?

Screenshot 2023-10-12 at 13 57 57

If you notice, the generated file is marked as "invalid symlink". 
I think the point is, you're providing a relative path as the `source destination` and, in that case, the working directory is the `link` location. 
Please try it in your local, or let me know if I've misunderstood something

Comment (caponetto) [reply .github/workflows/release_build.yml]: Got it, my bad! Thanks for insisting :)

Comment (yesamer) [reply .github/workflows/release_build.yml]: Np @caponetto , I did the exact same thing the first time I tackled this. Thanks again for solving it.

**Review by yesamer — APPROVED** [review]
👍
